src/components/FilterBar/FilterBar.stories.tsx

```diff
@@ -0,0 +1,495 @@
+        headingText="Row with Clear All and Submit butttons"
+        selectedItems={selectedItems}
+        isOpen={isModalOpen}
+        onToggle={onToggle}
+        onClear={onClearAll}
+        onSubmit={() => null}
+        showClearAll={true}
+        showSubmitAll={true}
+      >
+        <MultiSelectGroup
+          showLabel={false}
+          id="submit-button"
+          labelText="Submit Button"
+        >
+          {multiSelectItems &&
+            multiSelectItems.map((multiSelect, i) => {
+              if (i > 1) return;
+              return (
+                <MultiSelect
+                  key={`${multiSelect.id}-3`}
+                  id={`${multiSelect.id}-3`}
+                  label={multiSelect.name}
+                  variant="dialog"
+                  items={multiSelect.items}
+                  selectedItems={selectedItems}
+                  onChange={(e) => {
+                    onChange(e.target.id, `${multiSelect.id}-3`);
+                  }}
+                  onMixedStateChange={(e) => {
+                    return onMixedStateChange(
+                      e.target.id,
+                      `${multiSelect.id}-3`,
+                      multiSelect.items
+                    );
+                  }}
+                  onClear={() => {
+                    onClear(`${multiSelect.id}-3`);
+                  }}
+                  onApply={() => {}}
+                />
+              );
+            })}
+        </MultiSelectGroup>
+      </FilterBar>
+      <FilterBar
+        id="column-with-buttons"
+        headingText="Column with Clear All and Submit butttons"
+        selectedItems={selectedItems}
+        isOpen={isModalOpen}
+        onToggle={onToggle}
+        onClear={onClearAll}
+        onSubmit={() => null}
+        layout="column"
+        showClearAll={true}
+        showSubmitAll={true}
+      >
+        <MultiSelectGroup
+          showLabel={false}
+          id="submit-button"
+          labelText="Submit Button"
+        >
+          {multiSelectItems &&
+            multiSelectItems.map((multiSelect) => (
+              <MultiSelect
+                key={`${multiSelect.id}-3`}
+                id={`${multiSelect.id}-3`}
+                label={multiSelect.name}
+                variant="dialog"
+                items={multiSelect.items}
+                selectedItems={selectedItems}
+                onChange={(e) => {
+                  onChange(e.target.id, `${multiSelect.id}-3`);
+                }}
+                onMixedStateChange={(e) => {
+                  return onMixedStateChange(
+                    e.target.id,
+                    `${multiSelect.id}-3`,
+                    multiSelect.items
+                  );
+                }}
+                onClear={() => {
+                  onClear(`${multiSelect.id}-3`);
+                }}
+                onApply={() => {}}
+              />
+            ))}
+        </MultiSelectGroup>
+      </FilterBar>
```

Comment:
This is is not really an issue but because every `FilterBar` uses the same props when they are rendered, then when you select a checkbox in one MultiSelect (like in Colors), then it selects the same value for all other MultiSelects.
You can keep this as is but add a sentence or two explaining that this functionality only occurs for the Storybook example.